### ISSUE TITLE
Boards cannot be manufactured by Aisler

### ISSUE THREAD

**Issue by mthiesen** [Jartza/galpla-smd#1]
The current gerber files cannot be manufactured by [Aisler](https://aisler.net) because two drills are too close together.

![aisler_issue](https://user-images.githubusercontent.com/5121122/193011825-2eb9b26f-db8d-48d8-b2a1-6fe3a7301fac.png)

I wanted to fix this issue myself but the board in the EAGLE board file is routed completely differently and I do not want to work on an outdated board file. I presume this is an outdated version because drills are intersecting pads and the trace from pin 1 is crooked.

![eagle_board](https://user-images.githubusercontent.com/5121122/193011821-a673fd58-0471-42ea-9f9d-ab2e118335d7.png)

**Comment by Jartza** [comment]
I just noticed this, for some reason GitHub didn't notify me of new issue, I need to check my settings.

Indeed, I noticed also myself some time ago that I updated intermediate version instead of the updated version - the reason for update was people complaining how hard the previous one was to hand-solder, so I extended the pads a little and re-traced the board.

Both the old version (in PCBWAY) and the latest update (Oct 27) has been verified working.
